README.md

```diff
@@ -32,6 +32,15 @@ The same format is used for `allow` and `block` filter files. Create a file for
 norobod -allow path/to/allow.csv -block path/to/block.csv <other options>
 ```
 
+#### Exec Commands
+Allows users to execute external commands as a filter. Users can create an executable add-on that sends either `Block` or `Allow` to std.out to block or allow a call.
```

Comment:
Can you change the values returned via `stdout` to be all lowercase (`block` and `allow`) so it's consistent with the values returned by `norobod`'s `/calls` endpoint?